pkg/cmd/server/api/v1/conversions.go

```diff
@@ -19,6 +19,9 @@ func init() {
 			if obj.ServingInfo.RequestTimeoutSeconds == 0 {
 				obj.ServingInfo.RequestTimeoutSeconds = 60 * 60
 			}
+			if obj.ServingInfo.MaxRequestsInFlight == 0 {
+				obj.ServingInfo.MaxRequestsInFlight = 500
```

Comment:
remind me again which long-running requests get counted against this? watch, pods/logs, pods/proxy, pods/exec, builds/logs?